packages/agreement-process/src/services/readmodel/readModelService.ts

```diff
@@ -203,17 +203,12 @@ const getAllAgreements = async (
 async function getAttribute(
   attributes: AttributeCollection,
   filter: Filter<{ data: Attribute }>
-): Promise<WithMetadata<Attribute> | undefined> {
+): Promise<Attribute | undefined> {
   const data = await attributes.findOne(filter, {
     projection: { data: true, metadata: true },
```

Comment:
since we are no more using `metadata`, it may be cleaner to exclude it from the query (same for other occurrences)
```suggestion
    projection: { data: true, metadata: false },
```

Reply:
✔️  done!